tests/tcg/ppc64le/mffsce.c

```diff
@@ -0,0 +1,38 @@
+#include <stdlib.h>
+#include <stdint.h>
+#include <assert.h>
+
+#define MTFSF(FLM, FRB) asm volatile ("mtfsf %0, %1" :: "i" (FLM), "f" (FRB))
+#define MFFS(FRT) asm("mffs %0" : "=f" (FRT))
+#define MFFSCE(FRT) asm("mffsce %0" : "=f" (FRT))
+
+#define PPC_BIT_NR(nr) (63 - (nr))
+
+#define FP_VE (1ull << PPC_BIT_NR(56))
+#define FP_OE (1ull << PPC_BIT_NR(57))
+#define FP_UE (1ull << PPC_BIT_NR(58))
+#define FP_ZE (1ull << PPC_BIT_NR(59))
+#define FP_XE (1ull << PPC_BIT_NR(60))
+#define FP_NI (1ull << PPC_BIT_NR(61))
+#define FP_RN0 (1ull << PPC_BIT_NR(62))
+#define FP_RN1 (1ull << PPC_BIT_NR(63))
+
+int main(void)
+{
+    uint64_t frt, fpscr;
+    uint64_t last_8_bits = FP_VE | FP_UE | FP_ZE |
+                           FP_XE | FP_NI | FP_RN1;
+    MTFSF(0b11111111, last_8_bits); // set test value to cpu fpscr
```

Comment:
AFAIK, binary constants are a GCC extension and not standard C.

See https://en.cppreference.com/w/c/language/integer_constant - maybe in C23 they become available :)

Reply:
Hmmm. I actually just copied this from tests/tcg/ppc64le/mtfsf.c. I agree it is not standard C (AFAIK too), but there is another test using it. Maybe @alqotel knows more about this

Reply:
Just talked with @mferst and @alqotel here and this is ok: it's used in multiple places in the project already, and is supported by both GCC and Clang, the compilers QEMU support.

Reply:
Ok